django_saml2_auth/tests/test_user.py

```diff
@@ -121,6 +121,34 @@ def test_create_new_user_success(settings: SettingsWrapper):
     assert user.groups.get(name="users") == Group.objects.get(name="users")
 
 
+@pytest.mark.django_db
+def test_create_new_user_with_dict_success(settings: SettingsWrapper):
```

Comment:
Write another test to see if explicitly not passing `firstname` and `lastname` causes any issues or not.